app/src/main/java/com/microsoft/office365/connectmicrosoftgraph/AuthenticationManager.java

```diff
@@ -120,7 +112,8 @@ public String getAccessToken() {
      *
      * @param authenticationCallback The callback to notify when the processing is finished.
      */
-    public void connect(final AuthenticationCallback<AuthenticationResult> authenticationCallback) {
+    public void connect(final AuthenticationCallback<AuthenticationResult> authenticationCallback)
+            throws AuthenticationException {
```

Comment:
File an issue in the ADAL repo. Adding check exceptions should be a major change. It breaks previously working code.